src/osd/OSD.cc

```diff
@@ -8262,6 +8262,19 @@ void OSD::handle_osd_map(MOSDMap *m)
 	  << dendl;
 
   logger->inc(l_osd_map);
+  // since the map received counter is being incremented we know that the osd has recieved a map
+  // first check if it contains a full map in which case increase that logger count by 1 
```

Comment:
Mmm, we know the osd received a map because we're looking at the MOSDMap message we got, not because of the counter.  Also, we increment the logger by maps.size(), which may be > 1.  I'd argue that we probably don't need this comment.